fairscale/nn/__init__.py

```diff
@@ -3,23 +3,12 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
+from typing import List
+
 from .data_parallel import FullyShardedDataParallel, ShardedDataParallel
 from .misc import FlattenParamsWrapper, checkpoint_wrapper
 from .moe import MOELayer, Top2Gate
 from .pipe import Pipe, PipeRPCWrapper
 from .wrap import auto_wrap, default_auto_wrap_policy, enable_wrap, wrap
 
-__all__ = [
```

Comment:
just curious, what prevents existing import statements from breaking if we remove this?

Reply:
Nothing breaks. It simply changes the behavior of "import *", which is only used during debugging session etc. afaik.